keps/prod-readiness/template/nnnn.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 0000 
```

Comment:
Should this link to the alias so people know what the list is?

Reply:
Sure, will change here